app/src/main/java/org/mozilla/reference/browser/components/BackgroundServices.kt

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.reference.browser.components
+
+import android.content.Context
+import mozilla.components.browser.storage.sync.PlacesHistoryStorage
+import org.mozilla.reference.browser.services.AccountStateMachine
+import org.mozilla.reference.browser.services.Init
+import org.mozilla.reference.browser.services.SyncManager
+
+/**
+ * Component group for background services.
+ */
+class BackgroundServices(
+    context: Context,
+    placesHistoryStorage: PlacesHistoryStorage
+) {
+    val accountManager = AccountStateMachine(context).also { stateMachine ->
```

Comment:
Let's try to hide this so that API consumers only have to `start/stop` the FirefoxAccountsFeature. This could happen as part of `start`